.jshintrc

```diff
@@ -1,6 +1,4 @@
 {
-  "camelcase": true,
-  "curly": true,
```

Comment:
Just a quick note on why I made these changes. There are several cases where having single-line if statements without curly braces improve readability. The same goes for maxlen - there are cases where a line may go beyond 80 chars, but it still looks more readable than breaking it into two lines; though, I'm still a proponent of aiming for keeping lines at <= 80 chars. I also am of the philosophy of not wanting to sprinkle in jshint or instanbul ignore directives throughout code to further separate concerns.

Granted, these are personal opinions based on my personal preferences so I'm happy to adhere to whichever guidelines you'd like to enforce! :+1: